test/spec/components/Feel.spec.js

```diff
@@ -277,6 +277,29 @@ describe('<FeelField>', function() {
         expect(error.innerText).to.eql('error');
       });
 
+
+      it('should pass error to `setValue`', function() {
+
+        // given
+        const setValueSpy = sinon.spy();
+        const validate = (v) => {
```

Comment:
I know we use this pattern a lot, but these validation functions are pointless. We know exactly if the validation is to return an error or not. So why not just do `const validate = () => 'error'`? I'll file a follow-up pull request simplifying this. Our tests should stay focused.

Reply:
This might have been deliberate in the beginning (e.g. only set "invalid" when the check actually fails. However, we never do the `// assume - valid at the start` part, so we should be good to simplify